integration_tests/src/wallet_process.rs

```diff
@@ -143,6 +149,7 @@ pub async fn spawn_wallet(
         wallet_app_config.wallet.p2p.transport.transport_type = TransportType::Tcp;
         wallet_app_config.wallet.p2p.transport.tcp.listener_address =
             Multiaddr::from_str(&format!("/ip4/127.0.0.1/tcp/{}", port)).unwrap();
+        wallet_app_config.wallet.http_server_url = format!("http://127.0.0.1:{}", http_port);
```

Comment:
I run it locally like this to be sure it connects locally, otherwise I think it goes to the default fallback server
```suggestion
        wallet_app_config.wallet.http_server_url = format!("http://127.0.0.1:{}", http_port);
        wallet_app_config.wallet.fallback_http_server_url = format!("http://127.0.0.1:{}", http_port);
```